AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6615 @@
+{
```

Comment:
<p>I assume as you're coding this you'll explain the different lines/arguments?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Yes</p>